dev-scripts-additional-config

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+export ENABLE_LOCAL_REGISTRY=true
```

Comment:
To test in isolation just ENABLE_LOCAL_REGISTRY I'd suggest setting also:
```
export IP_STACK=v4
export NETWORK_TYPE=OpenShiftSDN
```

Otherwise IPv6 will be enabled by default, and thus also MIRROR_IMAGES